.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!-- Please take a look at our [Contributing](https://rook.io/docs/rook/master/development-flow.html)
+documentation before submitting a Pull Request!
+Thank you for contributing to Rook! -->
+
+Is this a fix for an issue or a feature implementation?
+<!-- Remove only one -->
+* Fix
+* Feature
+
+Description of your changes:
+
+Which issue is resolved by this Pull Request:
+
+Does your change break an existing feature:
+
+Checklist:
+- [ ] Read the [contribution guide](https://github.com/rook/rook/blob/master/CONTRIBUTING.md).
```

Comment:
I don't think the first two bullets are necessary. They are covered sufficiently by the comment on line 1. What about the following checklist?
- Documentation has been updated, if necessary
- PendingReleaseNotes.md has been updated with breaking and/or notable changes, if necessary